src/components/forgot-password/forgot-password-form.tsx

```diff
@@ -0,0 +1,84 @@
+import { useFormContext } from "react-hook-form"
+import Image from "next/image"
+import ErrorMessage from "@/components/error/error-message"
+import { EnvelopeIcon } from "@heroicons/react/24/outline"
+import Link from "next/link"
+import React from "react"
+import { resetPasswordForEmail } from "@/services/authentication.service"
+import { useSupabaseClient } from "@supabase/auth-helpers-react"
+
+export interface ForgotPasswordInputs {
+  email: string
+}
+
+interface ForgotPasswordFormProps {
+  setEmailIsSent: React.Dispatch<React.SetStateAction<boolean>>
+}
+
+export const ForgotPasswordForm = ({ setEmailIsSent }: ForgotPasswordFormProps) => {
+  const supabase = useSupabaseClient()
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useFormContext<ForgotPasswordInputs>()
+
+  const resetPassword = async ({ email }: ForgotPasswordInputs) => {
+    if (await resetPasswordForEmail(supabase, email)) setEmailIsSent(true)
+  }
+
+  return (
+    <div className={"auth-container text-center"} data-cy={"request-link"}>
+      <Image
+        src={"/images/undraw-forgot-password.svg"}
+        alt={"Person looking confused at password field"}
+        width={0}
+        height={0}
+        style={{ width: "100%" }}
```

Comment:
why the width 0 and then 100%?

Reply:
https://github.com/vercel/next.js/discussions/18474#discussioncomment-5501724